bin/acala/service/src/chain_spec/acala.rs

```diff
@@ -22,8 +22,11 @@ pub fn acala_config() -> Result<ChainSpec, String> {
 
 pub fn latest_acala_config() -> Result<ChainSpec, String> {
 	let mut properties = Map::new();
-	properties.insert("tokenSymbol".into(), "ACA".into());
-	properties.insert("tokenDecimals".into(), 13.into());
+	properties.insert(
+		"tokenSymbol".into(),
+		vec!["ACA", "AUSD", "DOT", "LDOT", "POLKABTC", "XBTC", "RENBTC", "PHA", "PLM"].into(),
```

Comment:
better to avoid hardcoded values here. it will be better to get them from CurrencyId directly